go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/libp2p/go-libp2p-peerstore
+
+require (
+	github.com/AndreasBriese/bbloom v0.0.0-20180913140656-343706a395b7 // indirect
+	github.com/Kubuxu/go-os-helper v0.0.0-20161003143644-3d3fc2fb493d // indirect
+	github.com/dgraph-io/badger v1.5.4 // indirect
+	github.com/dgryski/go-farm v0.0.0-20190104051053-3adb47b1fb0f // indirect
+	github.com/go-check/check v0.0.0-20180628173108-788fd7840127 // indirect
+	github.com/gogo/protobuf v1.2.1
+	github.com/google/uuid v1.1.1 // indirect
+	github.com/hashicorp/golang-lru v0.5.1
+	github.com/ipfs/go-datastore v0.0.0-20190207190025-057884927798
+	github.com/ipfs/go-ds-badger v0.0.0-20190218174214-7d5e050ec3cd
+	github.com/ipfs/go-ds-leveldb v0.0.0-20190207190115-d9761ea1cccd
+	github.com/ipfs/go-log v0.0.0-20190218133459-8924f37936b0
```

Comment:
Above 4 IPFS dependencies need to be gomodded and updated here with their v0.0.1 versions. cc @Kubuxu

Reply:
These 4 should be out.